app-ios/Makefile

```diff
@@ -51,9 +51,10 @@ test: ## Run Core tests (Linux-compatible)
 	@echo "Tests complete!"
 
 .PHONY: test-native
-test-native: ## Run Native module tests (macOS only)
-	@echo "Running Native tests (macOS only)..."
-	@cd Native && swift test --enable-experimental-prebuilts
+test-native: ## Run Native module tests (iOS Simulator)
+	@echo "Running Native tests on iOS Simulator..."
+	@xcodebuild test -scheme ComponentTests -destination 'platform=iOS Simulator,name=iPhone 16 Pro' -quiet
+	@xcodebuild test -scheme FeatureTests -destination 'platform=iOS Simulator,name=iPhone 16 Pro' -quiet
```

Comment:
Q: When I tried running this command, it failed because the FeatureTest scheme was missing. After checking Xcode, I found that although FeatureTest exists as a test target in my environment, the scheme itself doesn’t exist and needs to be added. Was this scheme supposed to be set up during an earlier setup phase, or was it accidentally left out of a commit?


<img width="802" height="470" alt="Image" src="https://github.com/user-attachments/assets/72e85432-26b8-4cbe-aa3d-6f6a4b39fe0e" />

Reply:
Sorry, I found this was an issue with my environment (because I manually added some schemes, Xcode failed to create schemes automatically), so I have marked this conversation as resolved.

I'll check this PR again!